tests/kubectl_test.go

```diff
@@ -37,6 +37,8 @@ var _ = Describe("[rfe_id:3423][vendor:cnv-qe@redhat.com][level:component]oc/kub
 		result, _, _ := tests.RunCommand(k8sClient, verb, resource)
 		Expect(result).ToNot(BeNil())
 		resultFields := strings.Fields(result)
+		// Verify that only Header is not present
+		Expect(resultFields).ShouldNot(Equal(expectedHeader))
```

Comment:
I don't understand how this is going to fix the test. Can you elaborate please?

Reply:
This is not fixing anything atm, this should just give us the output of `resultFields` if it's simply equal to `expectedHeader` instead of a plain panic on the next line

Reply:
Can you do something like `Expect(len(resultFields)).Should(BeNumerically(">=", len(expectedHeader)))`?

Reply:
Added, but `>` is better I think